test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -2580,7 +2587,17 @@ TEST_F(DataReaderTests, delete_contained_entities)
     // To be updated when Query Conditions are available
     ASSERT_EQ(query_condition, nullptr);
 
+    std::cout << "check non-recursive?" << std::endl;
+
+    // Should failed with outstanding ReadConditions
+    ASSERT_EQ(subscriber->delete_datareader(data_reader), ReturnCode_t::RETCODE_PRECONDITION_NOT_MET);
+
+    std::cout << "Before recursive?" << std::endl;
+
+    // Should not failed with outstanding ReadConditions
     ASSERT_EQ(data_reader->delete_contained_entities(), ReturnCode_t::RETCODE_OK);
+
+    std::cout << "Where is the SEH?" << std::endl;
```

Comment:
I hope it's nowhere to be seen. Please remove this trace :smile:

Reply:
done